README.md

```diff
@@ -1,40 +1,45 @@
 # Meowspace
-### Layout Practice!
 
+This is an individual, [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md) project.
 
-## Instructions
+## Learning Goals
+- Practice Layout Techniques
+- Have DRY, readable CSS code using CSS selectors with classes and ids
+- Add Custom Styling
 
-Use the drawings below to make the provided html code match the layouts. These drawings are called wireframes, they are like a blueprint for your site's layout design. It highlights the layout without focusing on any of the design specifics (like color, fonts etc.).
+## Baseline
+Using CSS, construct the layout of the site according to the drawing below.
+![Meowspace Wireframe](./assets/meowspace.png)
 
 
-First, use CSS to rearrange the layout to match the drawings as much as possible (it doesn't need to be perfect).  Only then can you jazz up the site's design with more pleasing colors, fonts and small touches.
+This drawings is called a wireframe. Wireframes are like a blueprint for your site's layout design. It highlights the layout without focusing on any of the design specifics (like color, fonts etc.). The areas that have an giant 'x' crossed through it are meant to represent the areas that have an image. **Hint:** There are two sections in this wireframe that have images as backgrounds.
 
 
-- You can make changes to the html
-  - WITHOUT moving any content around
-  - You can add sectioning elements, ID's and classes and necessary
+**You can make only make limited changes to the HTML**
+  - You cannot move any content around in the HTML
+  - You can add sectioning elements to wrap around content
+  - You can add ID's and classes to elements, as necessary
 
+## Primary Requirements
 
+- Add normalize.css  
```

Comment:
I was referring to this https://necolas.github.io/normalize.css/

I know i brought it up with C6, but I can through in a quick spiel about why it is and why to use it.